managed-ledger/src/main/java/org/apache/bookkeeper/mledger/ManagedLedgerConfig.java

```diff
@@ -45,7 +45,9 @@ public class ManagedLedgerConfig {
     private boolean createIfMissing = true;
     private int maxUnackedRangesToPersist = 10000;
     private int maxBatchDeletedIndexToPersist = 10000;
+    private String cursorInfoCompressionType = "NONE";
```

Comment:
Let's not break backwards compatibility

Reply:
isnt' it part of this new feature? then how come it will impact backward compatibility? if we will introduce it in this feature then we will have compatibility issue and that's what I would like to avoid by not adding it here,